modules/friendlywelcome.js

```diff
@@ -183,145 +183,41 @@ Twinkle.welcome.populateWelcomeList = function(e) {
 			type: 'radio',
 			name: 'template',
 			list: Twinkle.getFriendlyPref('customWelcomeList'),
-			event: Twinkle.welcome.selectTemplate
+			event: function() {
+				e.target.form.article.disabled = true;
```

Comment:
Hmm. Actually I didn't intend to make the change (and still ended up changing it for the better?)! I misread `Twinkle.welcome.selectTemplate` and thought that it disables the field for all custom tags. 

Looking at the code, the field's value (`article`) is being used as: `'{{subst:' + template + (article ? '|art=' + article : '') + '}}'` --  but obviously, this works only if the template supports an `|art=` parameter. 

It would have been great if we could know the usage syntax of custom template, but that's not possible without a big change to the twinkleconifg module. 

I think the right thing to do will be to change it back to how it was before (`false`) and think about it in a separate issue?

Reply:
Yes, that'd be better; I was mistaken on how we handled those, so yes, this needs to stay enabled for custom templates.

Reply:
changed to `false`.